iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -310,6 +337,24 @@ public void rollbackAlterTable(org.apache.hadoop.hive.metastore.api.Table hmsTab
     }
   }
 
+  private void setupAlterOperationType(EnvironmentContext context) throws MetaException {
+    if (context != null) {
+      Map<String, String> contextProperties = context.getProperties();
+      if (contextProperties != null) {
+        String stringOpType = contextProperties.get(ALTER_TABLE_OPERATION_TYPE);
+        if (stringOpType != null) {
+          currentAlterTableOp = AlterTableType.valueOf(stringOpType);
+          if (SUPPORTED_ALTER_OPS.stream().noneMatch(op -> op.equals(currentAlterTableOp))) {
+            throw new MetaException(
+                "Unsupported ALTER TABLE operation type for Iceberg tables, must be: " + allowedAlterTypes.toString());
+          }
+        }
+        return;
+      }
+    }
+    throw new MetaException("ALTER TABLE operation type could not be determined.");
```

Comment:
Can we maybe get rid of the return by putting this exception to the beginning of the method?
e.g. 
```
if (context == null || context.getProperties() == null) {
   throw new ...
}
```
The other thing I'm thinking of is that it'd be informative to include the hmsTable name in the exception message as well (for this and the above too).